src/perception/aruco_detect.hpp

```diff
@@ -70,25 +55,24 @@ class FiducialsNode {
     bool mPublishImages = false; // If set, we publish the images with the fiducials drawn on top
     bool mEnableDetections = true;
     bool mIsVerbose = false;
-    bool mHasCamInfo = false;
     bool mPublishFiducialTf = false;
-    double mFiducialLen{};
+    double mMinFramesSeen = 5;
     std::vector<int> mIgnoreIds;
     int mFilterCount{};
     double mFilterProportion{};
     cv::Ptr<cv::aruco::DetectorParameters> mDetectorParams;
     cv::Ptr<cv::aruco::Dictionary> mDictionary;
 
+    PointCloudPtr mCloudPtr = boost::make_shared<PointCloudPtr::element_type>();
```

Comment:
boost? Why not `std::shared_ptr`?

Reply:
Its required for the PCL library unfortunately

Reply:
it looks like they merged something to fix that a couple years ago: https://github.com/PointCloudLibrary/pcl/pull/3750. Can we update to a newer version of the lib?

Reply:
I think this is a limitation of ROS PCL. I'm against Boost too but the headers are already there.